operator/controllers/seldondeployment_controller.go

```diff
@@ -170,6 +170,46 @@ func createIstioResources(mlDep *machinelearningv1.SeldonDeployment,
 			return nil, nil, err
 		}
 	}
+
+	getDefaultHttpRoute := func() istio_networking.HTTPRoute {
```

Comment:
Why should we define it as a anon func here?

Reply:
Because we will get it as a base data, and will generate more HTTPRoute(for each Match rule, will generate a HTTPRoute structure).
Before this PR, have only a HTTPRoute, not need generate more than one HTTPRoute struct.

Reply:
SGTM